src/cli/install.sh

```diff
@@ -224,6 +224,35 @@ zsh)
         done
     fi
     ;;
+bash)
+    commands=(
+        "export $install_env=$quoted_install_dir"
+        "export PATH=\"$bin_env:\$PATH\""
+    )
+
+    bash_config=$HOME/.bashrc
```

Comment:
There isn't a standard for this

It should try many:
- .bash_profile
- .bashrc
- $XDG_CONFIG_HOME/.bash_profile
- $XDG_CONFIG_HOME/.bashrc
- $XDG_CONFIG_HOME/bash_profile
- $XDG_CONFIG_HOME/bashrc
- /usr/local/bashrc
- /usr/local/bash_profile 

There are probably many more. It also needs to handle permission checks. The chosen location may exist but lack write permissions.

Reply:
We shouldn't try /usr/local/bashrc &
/usr/local/bash_profile because these are system ones that are copied when the user is created.